demo-projects/meetup/site/helpers/index.js

```diff
@@ -1,8 +1,14 @@
 import { format, isFuture } from 'date-fns';
+import { breakpoints } from '../theme';
 
 // Check if date is in future or past
 export const isInFuture = date => isFuture(date);
 
 // Pretty date formatting
 export const formatFutureDate = date => format(date, 'ddd D MMM, h:mm A');
 export const formatPastDate = date => format(date, 'MMM YYYY');
+
+// Media queries getter
+export const getBreakpoints = () => {
+  return Object.values(breakpoints).map(bp => `@media (min-width: ${bp}px)`);
```

Comment:
Why is this a function? Why not export the breakpoints directly?

Reply:
very good question - because i am tired 😂 

This will go away eventually, Joss implemented a much better version of media helpers

Reply:
Ah, yeah, facepaint, that would definitely be better.